include/internal/libspdm_fips_lib.h

```diff
@@ -0,0 +1,32 @@
+/**
+ *  Copyright Notice:
+ *  Copyright 2023 DMTF. All rights reserved.
+ *  License: BSD 3-Clause License. For full text see link: https://github.com/DMTF/libspdm/blob/main/LICENSE.md
+ **/
+
+#if LIBSPDM_FIPS_MODE
+
+/**
+ * HMAC KAT test: HMAC-SHA512 KAT and HMAC-SHA384 KAT and HMAC-SHA256 KAT.
+ * HMAC-SHA512 KAT covers SHA384, SHA512, HMAC-SHA384, and HMAC-SHA512.
+ * HMAC-SHA384 KAT covers SHA384, SHA512, HMAC-SHA384, and HMAC-SHA512.
+ * HMAC-SHA256 KAT covers SHA1, SHA224, SHA256, HMAC-SHA1, HMAC-SHA224, and HMAC-SHA256.
```

Comment:
Only list the algorithms that will be covered. For example SPDM doesn't support SHA1.

Reply:
Thanks. I have made the comments more clear.